### PR TITLE
Merge getSkins method and cleanup some command stuff

### PR DESCRIPTION
**Pull request by AlexProgrammerDE**

### CODE REVIEW

File: shared/src/main/java/net/skinsrestorer/shared/interfaces/ISRPlugin.java [hunk 1]
```diff
@@ -44,7 +44,6 @@ public interface ISRPlugin {
 
     String getVersion();
 
-    MetricsCounter getMetricsCounter();
```
Comment (xknat) [inline shared/src/main/java/net/skinsrestorer/shared/interfaces/ISRPlugin.java:47]: intended?

Comment (AlexProgrammerDE) [reply shared/src/main/java/net/skinsrestorer/shared/interfaces/ISRPlugin.java:47]: yup. unused code

**Review by xknat — APPROVED** [review]
LGTM

**Review by xknat — APPROVED** [review]
LGTM

**Pull request merged**